racunaljka.js

```diff
@@ -0,0 +1,230 @@
+/*=================================== brojevi==================================================*/
+
+function seeAll() {
+document.getElementById('izvestaj').focus();
+}
+
+function brojJedan() {
+    var pise = document.getElementById('izvestaj');
+   pise.value += 1;
+}
+
+function brojDva() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 2;
+}
+
+function brojTri() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 3; 
+}
+
+function brojCetiri() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 4; 
+}
+
+function brojPet() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 5; 
+}
+
+function brojSest() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 6; 
+}
+
+function brojSedam() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 7; 
+}
+
+function brojOsam() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 8; 
+}
+
+function brojDevet() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 9; 
+}
+
+function brojNula() {
+    var pise = document.getElementById('izvestaj');
+    if(pise.value==="") {
+       pise.value='';
+    }
+    else{pise.value+=0}
+}
+/*=================================== brojevi==================================================*/
+/*=================================== operacije================================================*/
+
+function plus() {
+    var pise = document.getElementById('izvestaj');
+       
+        if(pise.value==='') {
```

Comment:
This if does not make any sense. If value is already '', why do you set it to the same value?